packages/calypso-products/src/plans-list.tsx

```diff
@@ -1297,6 +1297,59 @@ const getPlanJetpackCompleteDetails = (): IncompleteJetpackPlan => ( {
 	],
 } );
 
+const JETPACK_INCLUDES_REAL_TIME_BACKUP = translate( 'Real-time backups as you edit' );
+
+const JETPACK_T1_INCLUDES_CLOUD_STORAGE = translate( '10GB of cloud storage' );
+const JETPACK_T2_INCLUDES_CLOUD_STORAGE = translate( '1TB (1,000GB) of cloud storage' );
+
+const JETPACK_MONTHLY_INCLUDES_LOG_ARCHIVE = translate( '30-day activity log archive' );
+const JETPACK_YEARLY_INCLUDES_LOG_ARCHIVE = translate( '1-year activity log archive' );
+
+const JETPACK_MONTHLY_INCLUDES_RESTORES = translate(
+	'Unlimited one-click restores from the last 30 days'
+);
+const JETPACK_YEARLY_INCLUDES_RESTORES = translate(
+	'Unlimited one-click restores from the last 1 year'
+);
+
+const JETPACK_INCLUDES_REALTIME_MALWARE_SCANNING = translate(
+	'Real-time malware scanning and one-click fixes '
+);
+const JETPACK_COMPLETE_INCLUDES_ADDITIONAL = [
+	translate( 'VideoPress with 1TB of ad-free video hosting' ),
+	translate( 'Site Search up to 100k records' ),
+	translate( 'CRM Entrepreneur' ),
+];
+const JETPACK_SECURITY_INCLUDES_SPAM_PROTECTION = translate(
+	'Comment and form spam protection (10k API calls/mo)'
+);
+const JETPACK_COMPLETE_INCLUDES_SPAM_PROTECTION = translate(
+	'Comment and form spam protection (60k API calls/mo)'
+);
+
+const JETPACK_BENEFITS_BEST_IN_CLASS_SUPPORT: TranslateResult = translate(
+	'Best-in-class support from WordPress experts'
+);
```

Comment:
Instead of creating these many constants for re-usability, it's better to use the strings inline inside `getJetpackIncludesInfo` etc. even if the strings are repeated. Using constants sometimes can result in lost context and a developer have to look up those contacts if they want to know what the benefits are.